### PR TITLE
[FAB-16789] Fix peer join description in network key concept

### PR DESCRIPTION
**Pull request by sijocherian**
#### Type of change
- Documentation update

#### Description
Fix the description in doc: About peer joining a channel, per 
https://jira.hyperledger.org/browse/FAB-16789

#### Additional details
Reference doc fixed https://hyperledger-fabric.readthedocs.io/en/latest/network/network.html#peers-and-ledgers

### CODE REVIEW

File: docs/source/network/network.md [hunk 1]
```diff
@@ -298,9 +298,10 @@ add more peers to the channel.
 
 A key part of a P1's configuration is an X.509 identity issued by CA1 which
 associates P1 with organization R1. Once P1 is started, it can **join** channel
-C1 using the orderer O4. When O4 receives this join request, it uses the channel
-configuration CC1 to determine P1's permissions on this channel. For example,
-CC1 determines whether P1 can read and/or write information to the ledger L1.
+C1 using the orderer O4. The orderer O4 uses the channel configuration CC1
```
Comment (denyeart) [inline docs/source/network/network.md]: Thanks for the contribution. 
The prior sentence about 'using the orderer 04' is still incorrect, that part needs to be fixed as well.

Comment (sijocherian) [reply docs/source/network/network.md]: Thanks. Fixed by removing that sentence.

**Review by nikhil550 — DISMISSED** [review]

File: docs/source/network/network.md [hunk 2]
```diff
@@ -297,10 +297,10 @@ to access. We can think of L1 as being **physically hosted** on P1, but
 add more peers to the channel.
 
 A key part of a P1's configuration is an X.509 identity issued by CA1 which
-associates P1 with organization R1. Once P1 is started, it can **join** channel
-C1 using the orderer O4. When O4 receives this join request, it uses the channel
-configuration CC1 to determine P1's permissions on this channel. For example,
-CC1 determines whether P1 can read and/or write information to the ledger L1.
+associates P1 with organization R1. The orderer O4 uses the channel configuration
```
Comment (denyeart) [inline docs/source/network/network.md]:
```suggestion
associates P1 with organization R1. When an organization administrator joins peer P1 to channel C1, and the peer starts pulling blocks from the orderer O4, the orderer uses the channel configuration
```

**Review by denyeart — COMMENTED** [review]
These changes have fixed the misinformation, but now the action of joining a peer to a channel has dropped out :-)
My suggested change should finish this off...

File: docs/source/network/network.md (hunk 2 above)
Comment (sijocherian) [reply docs/source/network/network.md]: Thanks. I felt I was dropping some bit of information.
Looks much better now

**Review by denyeart — APPROVED** [review]

**Comment by denyeart** [comment]
Thanks!

**Pull request merged**